scripts/fiber/tests-failing.txt

```diff
@@ -48,6 +48,7 @@ src/renderers/shared/__tests__/ReactPerf-test.js
 
 src/renderers/shared/hooks/__tests__/ReactComponentTreeHook-test.js
 * can be retrieved by ID
+* works
```

Comment:
Why is this now failing? It seems like you didn't do anything to disable existing behavior.